accelerator/apis.c

```diff
@@ -214,6 +214,98 @@ status_t api_receive_mam_message(const iota_client_service_t* const service,
   return ret;
 }
 
+static status_t mam_send_bundle(const iota_config_t* const tangle,
+                                const iota_client_service_t* const service,
+                                bundle_transactions_t* const bundle) {
+  Kerl kerl;
+  kerl_init(&kerl);
+  bundle_finalize(bundle, &kerl);
+  transaction_array_t* out_tx_objs = transaction_array_new();
+  hash8019_array_p raw_trytes = hash8019_array_new();
+  iota_transaction_t* curr_tx = NULL;
+  flex_trit_t trits_8019[FLEX_TRIT_SIZE_8019];
+
+  BUNDLE_FOREACH(bundle, curr_tx) {
+    transaction_serialize_on_flex_trits(curr_tx, trits_8019);
+    hash_array_push(raw_trytes, trits_8019);
+  }
+
+  ta_send_trytes(tangle, service, raw_trytes);
+
+  hash_array_free(raw_trytes);
+  transaction_array_free(out_tx_objs);
+
+  return SC_OK;
+}
+
+status_t api_mam_send_message(const iota_config_t* const tangle,
+                              const iota_client_service_t* const service,
+                              char const* const payload,
+                              const bool last_packet) {
+  status_t ret = SC_OK;
+  mam_api_t mam;
+  bundle_transactions_t* bundle = NULL;
+  mam_psk_t_set_t psks = NULL;
```

Comment:
Remove above hardcoded psk and test if this works?

Reply:
in mam/api/api.c `static retcode_t mam_api_bundle_write_header(mam_api_t *const api, tryte_t const *const ch_id,
                                             tryte_t const *const ep_id, tryte_t const *const ch1_id,
                                             tryte_t const *const ep1_id, mam_psk_t_set_t psks,
                                             mam_ntru_pk_t_set_t ntru_pks, trint9_t msg_type_id,
                                             bundle_transactions_t *const bundle, trit_t *const msg_id)`

Reply:
This func ask psks as argment, so I don't think we could remove hardcoded psk

Reply:
I see. We will handle this in other PR.